js/ui/scheduler/tooltip_strategies/desktopTooltipStrategy.js

```diff
@@ -135,9 +136,7 @@ class TooltipManyAppointmentsBehavior extends TooltipBehaviorBase {
         return this.scheduler.option("dropDownAppointmentTemplate") === "dropDownAppointment";
     }
 
-    _onAppointmentDragStart(itemData, settings, $target) {
-        this.scheduler.hideAppointmentTooltip();
-
+    _onAppointmentDragStart(itemData, settings, e) {
```

Comment:
I suggest renaming the "e" parameter to the  "eventArgs".

Reply:
done